ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/extension/clustering/ejb/main/module.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2021, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="org.wildfly.extension.clustering.ejb">
+
+    <properties>
+        <property name="jboss.api" value="private"/>
+    </properties>
+
+    <resources>
+        <artifact name="${org.wildfly:wildfly-clustering-ejb-extension}"/>
+    </resources>
+
+    <dependencies>
+        <module name="java.xml"/>
+
+        <module name="org.infinispan"/>
+        <module name="org.jboss.as.clustering.common"/>
+        <module name="org.jboss.as.controller"/>
+        <module name="org.jboss.as.ee"/>
+        <module name="org.jboss.as.server"/>
+        <module name="org.jboss.as.web-common"/>
+        <module name="org.jboss.dmr"/>
+        <module name="org.jboss.logging"/>
+        <module name="org.jboss.metadata.common"/>
+        <module name="org.jboss.metadata.web"/>
+        <module name="org.jboss.msc"/>
+        <module name="org.jboss.staxmapper"/>
+        <module name="org.jboss.vfs"/>
+        <module name="org.wildfly.clustering.ee.spi"/>
+        <module name="org.wildfly.clustering.ejb.infinispan"/>
+        <module name="org.wildfly.clustering.ejb.spi"/>
+        <module name="org.wildfly.clustering.infinispan.client"/>
+        <module name="org.wildfly.clustering.infinispan.spi"/>
+        <module name="org.wildfly.clustering.service"/>
+        <module name="org.wildfly.clustering.spi"/>
```

Comment:
Could you justify the uses of the following modules as dependencies? A inspection of `org.wildfly:wildfly-clustering-ejb-extension` reveals they are not used, so I believe you could be adding more dependencies than needed:

* java.xml
* org.infinispan
* org.jboss.as.ee
* org.jboss.as.server
* org.jboss.as.web-common
* org.jboss.logging
* org.jboss.metadata.common
* org.jboss.metadata.web
* org.jboss.vfs
* org.wildfly.clustering.infinispan.client

That could affect the trimming capabilities by provisioning more dependencies than needed, so it is worth double-checking them; there could be details that are not revealed by inspecting only the artifact itself, so I'm just asking.

Reply:
Thanks, Yeray. I incorporated your changes.

Reply:
You're welcome @rachmatowicz , the module looks good to me now.